source/backend/dal/Repositories/AcquisitionFileRepository.cs

```diff
@@ -143,7 +143,9 @@ public PimsAcquisitionFile Update(PimsAcquisitionFile acquisitionFile)
             }
 
             this.Context.Entry(existingAcqFile).CurrentValues.SetValues(acquisitionFile);
-            this.Context.UpdateChild<PimsAcquisitionFile, long, PimsAcquisitionFilePerson>(p => p.PimsAcquisitionFilePeople, acquisitionFile.Internal_Id, acquisitionFile.PimsAcquisitionFilePeople.ToArray());
+            this.Context.UpdateChild<PimsAcquisitionFile, long, PimsAcquisitionFilePerson>(p => p.PimsAcquisitionFilePeople, acquisitionFile.Internal_Id, acquisitionFile.PimsAcquisitionFilePeople.ToArray(), true);
```

Comment:
I don't think you want to set true here, that would allow the system to modify the person, which is not desirable. I think in this case we are just updating the reference to the entities, not the entities themselves.

Reply:
Updated